platforms/documentation/docs/src/snippets/daemon-jvm/customized-task/groovy/build.gradle

```diff
@@ -1,5 +1,28 @@
 // tag::customized-daemon-jvm-task[]
-tasks.named('updateDaemonJvm') {
+tasks.named("updateDaemonJvm") {
     jvmVersion = JavaLanguageVersion.of(17)
 }
 // end::customized-daemon-jvm-task[]
+
+// tag::customized-plat-daemon-jvm-task[]
+tasks.named("updateDaemonJvm") {
+    def myPlatform = BuildPlatformFactory.of(
+        org.gradle.platform.Architecture.AARCH64,
+        org.gradle.platform.OperatingSystem.MAC_OS
+    )
+    toolchainPlatforms.add(myPlatform)
```

Comment:
❓ We have many open issues with multi valued properties. Maybe this should use `set` to not be possibly ambiguous.

Reply:
Fixed.